### PR TITLE
tried to improve the main layout and styling of the home-page, still …

### PR DESCRIPTION
**Pull request by luc-sauvage**
…things to be clarified

hey Morteza here's a first cleanup, following the designs in figma... there are a few things which we need to discuss

### CODE REVIEW

File: src/pages/home/home.css [hunk 1]
```diff
@@ -29,3 +36,33 @@
...
+  box-sizing: border-box;
+  border-radius: 16px;
+  padding: 0.8rem 1rem;
+  color: inherit;
+  text-decoration: none;
+
+  /* HOVER ? */
+}
+
+.button-not-button {
```
[the remark this answers is not in the document]
Comment (luc-sauvage) [reply src/pages/home/home.css:52]: a div which looks like a button but it's not... but yes you are right, i will change it!

File: src/pages/home/Home.js [hunk 2]
```diff
@@ -45,33 +44,42 @@ class Home extends Component {
...
             {homeLocales.mission_1[lang]} <br /> {homeLocales.mission_2[lang]}
           </H3>
+          <Link className="mission-button" to="/aboutus">Learn more</Link>
         </Mission>
-        <Why/>
+        <Why />
         <How id="how-it-works">
-          <H2  style={{ textAlign: "center" }}>{homeLocales.how[lang]}</H2>
-          <HowGrid>
+          <H2 titleColor="white" style={{ textAlign: "center", marginBottom: "1.5rem" }}>{homeLocales.how[lang]}</H2>
```
[the remark this answers is not in the document]
Comment (luc-sauvage) [reply src/pages/home/Home.js:54]: i will!

[the remark this answers is not in the document]
Comment (luc-sauvage) [reply src/pages/home/Home.js:54]: done

File: src/pages/home/home.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (luc-sauvage) [reply src/pages/home/home.css:52]: done! I used `.category-title`

File: src/pages/home/home.css [hunk 3]
```diff
@@ -1,12 +1,12 @@
 /* home */
 
 .home-component {
-  background-image: url("../../assets/images/header_image.png");
-  background-repeat: no-repeat;
+  background-image: url("../../assets/images/backgrounds/hero-home-background-picture.jpg");
+  background-position: 0 120%;
```
[the remark this answers is not in the document]
Comment (luc-sauvage) [reply src/pages/home/home.css:5]: i set `background-position: bottom` here

File: src/pages/home/Home.js [hunk 4]
```diff
@@ -45,33 +44,42 @@ class Home extends Component {
...
-            <Card2 title="#5" text={homeLocales.step_5[lang]} />
-            <Card2 title="#6" text={homeLocales.step_6[lang]} />
-          </HowGrid>
+          </HowFlex>
         </How>
         <SecPanel>
           <Card3 text={homeLocales.opinion[lang]} source={card3[0].source} />
         </SecPanel>
-        <CommunityPanel id="community">
+        {/* <CommunityPanel id="community">
```
[the remark this answers is not in the document]
Comment (luc-sauvage) [reply src/pages/home/Home.js:75]: done!

File: src/components/styled.js [hunk 5]
```diff
@@ -115,14 +115,18 @@ const Card2Root = styled.div`
   display: flex;
   flex-direction: column;
   background: #ffffff;
-  /* non-active */
   border: 0.5px solid #baafa1;
   box-sizing: border-box;
   border-radius: 10px;
   padding: 0.15rem;
-  max-width: 170px;
+  margin: 1rem;
```
Comment (nancologist) [inline src/components/styled.js:122]: Please use pixel unit (px).

File: src/components/styled.js [hunk 6]
```diff
@@ -115,14 +115,18 @@ const Card2Root = styled.div`
...
   padding: 0.15rem;
-  max-width: 170px;
+  margin: 1rem; 
   align-items: center;
   box-sizing: border-box;
+  height: 20rem;
+  width: 18rem;
+  min-height: 20rem;
+  min-width: 18rem;
+
```
Comment (nancologist) [inline src/components/styled.js:129]: please use pixel units

File: src/components/styled.js [hunk 7]
```diff
@@ -178,63 +183,39 @@ const Card2Text = styled.div`
...
-  /* non-active */
   border: 0.2px solid #baafa1;
   box-sizing: border-box;
-  width: 50%;
-  height: 60%;
-  /* neun */
+  border-radius: 24px;
+  height: 240px;
+  width: 560px;
+  padding: 1rem 1.5rem;
```
Comment (nancologist) [inline src/components/styled.js:191]: pixel units please

File: src/components/styled.js [hunk 8]
```diff
@@ -178,63 +183,39 @@ const Card2Text = styled.div`
...
-    border-radius: 24px;
-    height: 240px;
-    width: 560px;
-    /* top: 680px;
-    left: 12rem; */
-    padding: 1rem 1.5rem;
-    margin-top: 20rem;
+  top: 87%;
+  left: -9%;
+  padding: 2rem;
```
Comment (nancologist) [inline src/components/styled.js:197]: please use px

File: src/pages/home/styled.js [hunk 9]
```diff
@@ -28,15 +28,21 @@ export const MainRow = styled.div`
 `;
 
 export const SecPanel = styled.div`
-  background-image: url("./image/learning.png");
-  height: 200px;
+  background-image: url("./image/playstation.jpg");
```
Comment (nancologist) [inline src/pages/home/styled.js:31]: Please add your picture to the ``assets/`` and then import the picture in this js file and then use ``${}`` as argument for url()

File: src/pages/home/home.css [hunk 10]
```diff
@@ -1,12 +1,12 @@
 /* home */
 
 .home-component {
-  background-image: url("../../assets/images/header_image.png");
-  background-repeat: no-repeat;
+  background-image: url("../../assets/images/backgrounds/hero-home-background-picture.jpg");
```
Comment (nancologist) [inline src/pages/home/home.css:4]: please import the picture and then use the name of variable here in a ${} string interpolation

**Review by nancologist — CHANGES_REQUESTED** [review]

**Pull request merged**